test/manifest.js

```diff
@@ -242,47 +242,55 @@ module.exports = [
   {
     example: "doc_inspector_basic.html",
     script: "inspector-01.js",
+    pending: true,
     targets: ["gecko"],
   },
   {
     example: "doc_inspector_basic.html",
     script: "inspector-02.js",
+    pending: true,
     targets: ["gecko"],
   },
   {
     // This  test just times out, it doesn't even seem to run
     example: "doc_inspector_styles.html",
+    pending: true,
     script: "inspector-03.js",
     targets: ["gecko"],
-    pending: true,
   },
   {
     example: "doc_inspector_styles.html",
+    pending: true,
     script: "inspector-04.js",
     targets: ["gecko"],
   },
   {
     example: "doc_inspector_sourcemapped.html",
+    pending: true,
     script: "inspector-05.js",
     targets: ["gecko"],
   },
   {
     example: "doc_inspector_shorthand.html",
+    pending: true,
     script: "inspector-06.js",
     targets: ["gecko"],
   },
   {
     example: "doc_inspector_shorthand.html",
+    pending: true,
     script: "inspector-07.js",
     targets: ["gecko"],
   },
   {
     example: "doc_inspector_basic.html",
+    pending: true,
     script: "highlighter.js",
     targets: ["gecko"],
   },
   {
     example: "doc_stacking.html",
+    pending: true,
```

Comment:
This is a lot of pending tests, but as long as we revisit before shipping the inspector that is okay

Reply:
Yeah. This one is *so* weird. More details here: https://www.notion.so/replayio/Elements-Panel-Doesn-t-Load-in-Test-1a7e7f06335b421c87421005eecb96ef